### ISSUE TITLE
the activation fn of depthwiseconv in res connectionis different from that of tensorflow version??

### ISSUE THREAD

**Issue by hurri2000** [iamhankai/ghostnet.pytorch#21]
in your tensorflow version ,activation = BNNoReLU

```
res = DepthConv(end_point, net, conv_def.kernel, stride=layer_stride, 
                                        data_format='NHWC', activation=BNNoReLU)
```
but in pytorch version, relu = True

`            self.shortcut = nn.Sequential(
                depthwise_conv(inp, inp, 3, stride, relu=True),
                nn.Conv2d(inp, oup, 1, 1, 0, bias=False),
                nn.BatchNorm2d(oup),`

**Comment by iamhankai** [comment]
Thanks for your remind. I'll fix it soon. The tensorflow version is the standard version (https://github.com/iamhankai/ghostnet).

**Comment by iamhankai** [comment]
In addition, there is another bug: the kernel size in `self.shortcut = nn.Sequential( depthwise_conv(inp, inp, 3, stride, relu=True), nn.Conv2d(inp, oup, 1, 1, 0, bias=False), nn.BatchNorm2d(oup),` should be set as `kernel_size`.

**Comment by glenn-jocher** [comment]
@iamhankai has this bug https://github.com/iamhankai/ghostnet.pytorch/issues/21#issuecomment-607164839 been fixed in the code? Last update to ghost_net.py says two months ago but your comment is more recent. 

If not can you point to where to make the change?

**Comment by glenn-jocher** [comment]
@hurri2000 @iamhankai I've submitted https://github.com/iamhankai/ghostnet.pytorch/pull/23 to fix https://github.com/iamhankai/ghostnet.pytorch/issues/21#issue-591768407,

For the kernel size replacement, do you mean 
https://github.com/iamhankai/ghostnet.pytorch/blob/2c90e67d8c33c44ec1bad12c9686f645b0d4de08/ghost_net.py#L101

should be this?
`depthwise_conv(inp, inp, kernel_size, stride, relu=False)`

**Comment by glenn-jocher** [comment]
@hurri2000 @iamhankai I've updated both changes in #23. @iamhankai can you review please?
<img width="1165" alt="Screen Shot 2020-04-10 at 12 49 37 PM" src="https://user-images.githubusercontent.com/26833433/79018900-d07dc380-7b29-11ea-9703-d03ede441407.png">

Also @iamhankai are there any other bug fixes or changes needed that you know of? Thank you!!

**Comment by iamhankai** [comment]
Thanks @glenn-jocher I merged your pull request.
